website/cue/reference/components/sinks.cue

```diff
@@ -644,7 +644,9 @@ components: sinks: [Name=string]: {
 					title: "Transport Layer Security (TLS)"
 					body:  """
 						Vector uses [OpenSSL](\(urls.openssl)) for TLS protocols due to OpenSSL's maturity. You can
-						enable and adjust TLS behavior using the [`tls.*`](#tls) options.
+						enable and adjust TLS behavior via the [`tls.*`](#tls) options and/or via the 
+						[OpenSSL configuration options](\(urls.openssl_conf)) exposed through environment variables, 
+						particularly `OPENSSL_CONF`.
```

Comment:
Similarly, I think that the wording here (and others below) is vague, misleading and confusnig.
Maybe a better wording could be:
```
enable and adjust TLS behavior via the [`tls.*`](#tls) options and/or via an
[OpenSSL configuration file](\(urls.openssl_conf)) specified using the
`OPENSSL_CONF` environment variable.
```